### ISSUE TITLE
provide alone code action switch

### ISSUE THREAD

**Issue by tjx666** [zardoy/vscode-fix-all-json#16]
So I can use like this：

```json
    "editor.codeActionsOnSave": {
        "source.fixAll": true
        "source.fixAll.jsonc": true
    }
```

**Comment by zardoy** [comment]
Probably makes sense, though the only extension that I know is using it is eslint.

On the other hand, if want to disable auto fixing for json only, you can use language specific setting:
```json
"[json][jsonc]":{
"editor.codeActionsOnSave": {
        "source.fixAll": false
    }
}
```

**Comment by zardoy** [comment]
Probably makes sense, though the only extension that I know is using it is eslint.

On the other hand, if want to disable auto fixing for json only, you can use language specific setting:
```json
"[json][jsonc]":{
"editor.codeActionsOnSave": {
        "source.fixAll": false
    }
}
```

**Comment by tjx666** [comment]
many:

<img width="516" alt="image" src="https://user-images.githubusercontent.com/41773861/214842889-d184ad88-4a93-4f76-9a01-e0e40423593b.png">

**Comment by tjx666** [comment]
Thanks for create so many awesome extensions!

**Comment by tjx666** [comment]
I'm exploring your extensions:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/214843137-2e071bb0-0e24-4095-82fa-417b4761af3b.png">

**Comment by tjx666** [comment]
I see that vscode has builtin auto fix for example: add quotes.
What's the difference between this extension and vscode builtin?

**Comment by zardoy** [comment]
> I see that vscode has builtin auto fix for example: add quotes.

Are you sure this code action coming not from [this extension](https://github.com/zardoy/vscode-fix-all-json/blob/4ed6bf99b0c69048353535dc595a2ec0716078aa/src/extension.ts#L111)? I wasn't aware of builtin code actions, will check later.

> I'm exploring your extensions:

Will have even more! I'm very slowly moving features from https://github.com/zardoy/vscode-experiments/tree/main/src/features when I have nothing to do :)

**Comment by tjx666** [comment]
Oh, I find it is fixed by prettier...

**Comment by zardoy** [comment]
> I see that vscode has builtin auto fix for example: add quotes.

I checked in vscode.dev and there is no builtin codeactions

**Comment by zardoy** [comment]
I'm adding a way to disable it with following (https://github.com/zardoy/vscode-fix-all-json/pull/17):

```json
"editor.codeActionsOnSave": {
    "source.fixAll": true,
    "source.fixAll.fixAllJson": false
},
```

But enable e.g. when `"source.fixAll": false` this setting won't affect as general IDE feature id disabled.

**Comment by tjx666** [comment]
<img width="484" alt="image" src="https://user-images.githubusercontent.com/41773861/215266755-f24d3dbd-e5a5-4d41-8aa5-37fd61b6be8d.png">

I see eslint can enable in this case.
